src/components/header/renderer.tsx

```diff
@@ -33,6 +33,7 @@ const formatExampleName = (name: string) => {
 class Header extends React.Component<Props, State> {
   private refGistForm: HTMLFormElement;
   private examplePortal = React.createRef<HTMLDivElement>();
+  private _listnerAttached: boolean = false;
```

Comment:
Refer to [this](https://github.com/tc39/proposal-class-fields/blob/master/PRIVATE_SYNTAX_FAQ.md), `#` is used for declaring private variables instead of `_`. Try this if that doesn't pass the test, name what the tests are suggesting.